src/dymaptic.GeoBlazor.Core/Scripts/jsBuilder.ts

```diff
@@ -727,9 +731,9 @@ export function buildJsQuery(dotNetQuery: DotNetQuery): Query {
         where: dotNetQuery.where ?? "1=1",
         spatialRelationship: dotNetQuery.spatialRelationship as any ?? "intersects",
         distance: dotNetQuery.distance ?? undefined,
-        units: dotNetQuery.units as any ?? null,
+        units: dotNetQuery.units as any ?? undefined,
         returnGeometry: dotNetQuery.returnGeometry ?? false,
-        outFields: dotNetQuery.outFields ?? null,
```

Comment:
Same here, default value is null:
https://developers.arcgis.com/javascript/latest/api-reference/esri-rest-support-Query.html#outFields

Reply:
Yes, I haven't actually improved anything here, but I also didn't hurt anything. I think I made this change as part of an investigation into an issue, and it seemed to make more sense to me to keep it `undefined` as more consistent.